### DISCUSSION TITLE
Command-line app for namae

### DISCUSSION THREAD

**Discussion by clemenzi** [uetchy/namae#171] [General]
I really like the idea of namae, I wanted to propose namae in a terminal version. I wanted to ask if I had your permission. Thanks! (Obviously I will insert repo links, url of the site etc....)

**Comment by uetchy** [comment]
I love the idea! I might have to implement the API key management concept with rate limit first to avoid the API limit kicking in on the APIs namae relies on. 

@aThebigbot Reply here if you are still interested in developing CLI for namae. I'll let you know when the user-facing API is publicly available!

**Reply by clemenzi** [reply]
Yes I'm still interested, if you want you can contact me on discord for simplicity (Thebigbot#2222).

**Reply by clemenzi** [reply]
If you prefer to talk by email or another social tell me, because communicating through this discussion is not very comfortable

**Reply by Madonnaaa** [reply]
Giulioendcodeeeddtjrkdbfjd

**Comment by Madonnaaa** [comment]
GiulioEndSecsi
